tic_tac_toe/game.py

```diff
@@ -87,18 +119,42 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    
```

Comment:
Just a note, the `elif`'s here could have just been if's and the `else` is unnecessary. Using `raise` will cause your code to break out of what it is doing (unless you're in a `try` block) much like it would if you were calling `return`.